history/cultures/annihilan.txt

```diff
@@ -15,6 +15,9 @@
 	discover_innovation = innovation_plenary_assemblies 
 	discover_innovation = innovation_crop_rotation 
 	#
+	discover_innovation = innovation_taming_the_skies
```

Comment:
I made it so they cant have light aerial,  this would solve the continuity issue were massive creatures would be riding small aerial,  like people riding dogs.

Reply:
> I made it so they cant have light aerial, this would solve the continuity issue were massive creatures would be riding small aerial, like people riding dogs.

Ok, but do you plan to fix the bug?

Reply:
If by fix you mean get rid of the tech in history for the massive creatures, then yes I have removed it.